webpack.config.js

```diff
@@ -71,7 +71,9 @@ const commonSettings = {
         WEBPAC_BASE_URL: JSON.stringify(process.env.WEBPAC_BASE_URL),
         FEATURES: JSON.stringify(process.env.FEATURES),
         SHEP_BIBS_LIMIT: JSON.stringify(process.env.SHEP_BIBS_LIMIT),
-        SIERRA_UPGRADE_AUG_2023: JSON.stringify(process.env.SIERRA_UPGRADE_AUG_2023)
+        SIERRA_UPGRADE_AUG_2023: JSON.stringify(process.env.SIERRA_UPGRADE_AUG_2023),
```

Comment:
I don't think this app actually needs this variable. I'm pretty sure it is only accessed by My Account in DFE.

Reply:
That's a good point. Whatever code was triggered by the `SIERRA_UPGRADE_AUG_2023` var should be the default in the RC implementation of My Account, making the env var unneeded.